<?xml version="1.0"?>
<launch>

	<arg name="id_robot" default="rb1_base_a"/>
	<param name="ROBOT_ID" type="str" value="$(arg id_robot)"/>

	<group ns="$(arg id_robot)">
		<node pkg="fms_rob" name="test" type="test.py" output="screen"/>			
	</group>
	
</launch>
